packages/eslint-config-cozy-app/package.json

```diff
@@ -19,17 +19,16 @@
     "vue.js"
   ],
   "dependencies": {
-    "@babel/eslint-parser": "^7.17.0",
-    "@typescript-eslint/eslint-plugin": "^5.19.0",
-    "@typescript-eslint/parser": "^5.19.0",
-    "eslint": "^8.13.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-promise": "^6.0.0",
-    "eslint-plugin-react": "^7.29.4",
-    "eslint-plugin-react-hooks": "^4.5.0",
-    "eslint-plugin-vue": "^8.6.0",
```

Comment:
isn't it a breaking change to remove plugin vue ?

Reply:
@trollepierre I guess it is, but do we even have a Vue app? It seems legacy

Reply:
@trollepierre added breaking change in commit description

Reply:
I wonder if we should ask to front team about their opinion. 
With a breaking change, there is no way we are faulty